docker/ubuntu-indexer/bin/system-setup.sh

```diff
@@ -23,7 +23,9 @@ apt_packages+=('libclang-dev')
 apt_packages+=('llvm')
 apt_packages+=('llvm-dev')
 apt_packages+=('npm')
-apt_packages+=('nodejs')
+
+# Dependency for nodejs
+apt_packages+=('apt-transport-https')
```

Comment:
Does this install curl? I see it depends on libcurl, but it's not clear it makes the client program appear.